x-pack/plugins/cases/server/client/cases/create.ts

```diff
@@ -71,6 +72,10 @@ export const create = async (
     );
   }
 
+  if (query.tags.some(isEmptyString)) {
+    throw Boom.badRequest('A tag must not be empty');
```

Comment:
Maybe change this to be `throw Boom.badRequest('A tag must contain non-space characters');`

Reply:
Is this message displayed to the user in the UI? If so, can it be internationalized?

Reply:
> Is this message displayed to the user in the UI? If so, can it be internationalized?

Yeah they are :/ we need to go through all of the cases backend and internationalize them all. I think we have an issue for that, let me find it.